### ISSUE TITLE
Ruby 3.2 support

### ISSUE THREAD

**Issue by Birdie0** [jasonfb/hot-glue#40]
Generator script uses `File.exists?`, which was removed in `3.2.0`, because of that it doesn't work properly on higher versions.

```
WARNING: error writing to config/hot_glue.yml --- undefined method `exists?' for File:Class
WARNING: error writing to spec/support/capybara_login.rb --- undefined method `exists?' for File:Class
```

Considerable solution here would be switching to `File.exist?` or `File.file?`.

**Comment by jasonfb** [comment]
thank you @Birdie0 I actually just found this myself. 

It is now fixed on `main` brain. Will be fixed (and in the release notes) on the next release

**Comment by jasonfb** [comment]
fixed in v0.5.7
